src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -60,7 +60,7 @@ const AclInput = ({ value, onChange, disabled, maxAccessLevel, autoFocus, ...pro
         ...props
       })
     ]),
-    div({ style: { marginLeft: '1rem' } }, [
+    !isAzureWorkspace && div({ style: { marginLeft: '1rem' } }, [
```

Comment:
Do we want to hide both of these checkboxes if `!isAzureWorkspace` or just the can-compute checkbox? The `switchCase` above makes me think we only want to suppress the can-computer checkbox.

Reply:
For now, only owners will be able to share Azure workspaces, and they won't have the option to grant can-share or can-compute permissions for readers or writers on Azure workspaces

The `switchCase` above is because I wanted the default values for can-compute and can-share to be false for Azure workspaces. Since the default value for can-compute for Google workspaces is true, I added the `switchCase` for Azure workspaces. A potentially better solution would be to update Rawls to ignore these values for Azure workspaces rather than caring about what the default value is in the UI.